op-service/eth/supervisor_status.go

```diff
@@ -15,6 +15,7 @@ type SupervisorSyncStatus struct {
 type SupervisorChainSyncStatus struct {
 	// LocalUnsafe is the latest L2 block that has been processed by the supervisor.
 	LocalUnsafe BlockRef `json:"localUnsafe"`
+	Unsafe      BlockID  `json:"unsafe"`
```

Comment:
Please change this to be explicitly `CrossUnsafe`

In other contexts, in geth RPC specifically, `unsafe` is synonymous with `latest`, which is the `local-unsafe` block.

We should probably document the format of the sync-status object in the RPC specs too.

The RPC method is already defined here: https://github.com/ethereum-optimism/specs/blob/main/specs/interop/supervisor.md#supervisor_syncstatus
Just needs a type definition for the returned status object.